modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.cc

```diff
@@ -552,25 +531,19 @@ LBSSolver::BoundaryOptionsBlock()
 
   params.SetGeneralDescription("Set options for boundary conditions. See \\ref LBSBCs");
   params.SetDocGroup("LBSUtilities");
-
   params.AddRequiredParameter<std::string>("name",
                                            "Boundary name that identifies the specific boundary");
   params.AddRequiredParameter<std::string>("type", "Boundary type specification.");
-
   params.AddOptionalParameterArray<double>("group_strength",
                                            {},
-                                           "Required only if `type` is `\"isotropic\"`. "
-                                           "An array of isotropic strength per group");
-
-  params.AddOptionalParameter(
-    "function_name",
-    "",
-    "Text name of the lua function to be called for this boundary condition."
-    "For more on this boundary condition type.");
-
+                                           "Required only if `type` is `\"isotropic\"`. An array "
```

Comment:
I would suggest to use either \` or `"` for the 'isotropic' type.

Reply:
Fixed.